travis/build_info.py

```diff
@@ -104,23 +122,67 @@ def repo_active_build_count(builds):
     return build_count, started_count
 
 
-def repo_active_job_count(active_builds):
+def get_job_counts(org):
+    """
+    Total job counts (active and total) for
+    an org
+    """
+    total_job_count = 0
+    total_started_job_count = 0
+
+    repos = get_repos(org)
+    for repo in repos:
+        repo_builds = get_active_builds(org, repo)
+        repo_jobs = 0
+        repo_started_jobs = 0
+        for build in repo_builds:
+            build_jobs = get_active_jobs(build['id'])
+            total, started = active_job_counts(build_jobs)
+            total_job_count += total
+            total_started_job_count += started
+            repo_jobs += total
+            repo_started_jobs += started
+        logger.debug("----> " + repo)
```

Comment:
Yeah I was mainly going for readability, esp with all the debug statements that come later.